apps/etterlatte-tilbakekreving/src/main/kotlin/tilbakekreving/config/ApplicationProperties.kt

```diff
@@ -50,12 +51,8 @@ data class ApplicationProperties(
                     behandlingScope = value("ETTERLATTE_BEHANDLING_SCOPE"),
                     proxyUrl = value("ETTERLATTE_PROXY_URL"),
                     proxyScope = value("ETTERLATTE_PROXY_SCOPE"),
-                    devMode = valueOrNull("DEV_MODE").toBoolean(),
+                    devMode = get("DEV_MODE").toBoolean(),
```

Comment:
Sånn som jeg leser det, hvis "DEV_MODE" eller "HTTP_PORT" ikke er definert kaster vi nå en feilmelding i stede for å sette False, er det meningen? Blir det håndtert?

Reply:
Hm, eg trur det ikkje skal vera funksjonell endring her - altså at get returnerer `String?` på samme måte som `valueOrNull` gjorde.

Reply:
Du har rett